apps/vs-code-designer/src/app/utils/codeless/connection.ts

```diff
@@ -14,8 +14,10 @@ import { addNewFileInCSharpProject } from './updateBuildFile';
 import { HTTP_METHODS, isString } from '@microsoft/logic-apps-shared';
 import type { ParsedSite } from '@microsoft/vscode-azext-azureappservice';
 import { nonNullValue } from '@microsoft/vscode-azext-utils';
+import { JwtTokenHelper, JwtTokenConstants } from '../../../../../vs-code-react/src/app/designer/services/JwtHelper';
```

Comment:
This import shouldn't be like this. Lets move the `JwtHelper` class to a library so it can be used in `vs-code-react` and `vs-code-designer` porjects. You can move the whole file to this path `libs/vscode-extension/src/lib/services`. Make sure to export it and import it in both projects and also add unit tests